templates/telemetry/normandy-login-study/normandy-login-study.4.schema.json

```
@@ -0,0 +1,100 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "normandy-login-study",
+  "properties": {
+    @TELEMETRY_ID_1_JSON@,
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_APPLICATION_1_JSON@,
+    "version": {
+      "type": "number",
+      "minimum": 4,
+      "maximum": 4
+    },
+    "payload" : {
+      "type": "object",
+      "properties": {
+        "password_manager_enabled": {
+          "type": ["boolean", "null"]
+        },
+        "browser_privatebrowsing_autostart": {
+          "type": ["boolean", "null"]
+        },
+        "network_cookie_lifetime_policy": {
+          "type": ["number", "null"]
+        },
+        "privacy_sanitize_sanitize_on_shutdown": {
+          "type": ["boolean", "null"]
+        },
+        "privacy_clear_on_shutdown_cookies": {
+          "type": ["boolean", "null"]
+        },
+        "network_cookie_cookie_behavior": {
+          "type": ["number", "null"]
+        },
+        "browser_startup_page": {
+          "type": ["number", "null"]
+        },
+        "session_days_old": {
+          "type": ["number", "null"]
+        },
+        "profile_days_old": {
+          "type": ["number", "null"]
+        },
+        "logins_accounts": {
+          "type": ["boolean", "null"]
+        },
+        "logins_accounts_uses_per_month": {
+          "type": ["number", "null"]
+        },
+        "google_accounts_cookie_present": {
+          "type": ["boolean", "null"]
+        },
+        "google_accounts_cookie_days_old": {
+          "type": ["number", "null"]
+        },
+        "has_allow_cookie_exceptions": {
+          "type": ["boolean", "null"]
+        },
+        "has_block_cookie_exceptions": {
+          "type": ["boolean", "null"]
+        },
+        "cookies_oldest_days_old": {
+          "type": ["number", "null"]
+        },
+        "activeAddons": {
+          "type": ["array", "null"],
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string"
+              },
+              "name": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "default_search_engine_is_google": {
+          "type": ["boolean", "null"]
+        },
+        "default_private_search_engine_is_google": {
+          "type": ["boolean", "null"]
+        },
+        "accounts_days_visited_per_month": {
+          "type": ["number", "null"]
+        },
+        "has_browser_search_with_ads": {
+          "type": ["boolean", "null"]
+        },
+        "has_browser_search_ad_clicks": {
+          "type": ["boolean", "null"]
+        },
+        "history_oldest_days_old": {
+          "type": ["number", "null"]
+        }
```

```suggestion
        },
        "history_average_days_per_month": {
          "type": ["number", "null"]
        }
```

This field is present in the example, but not the schema. It would go to additional_properties in BQ if not in the schema.